docs/user_guide/specifying_finance_parameters.md

```diff
@@ -0,0 +1,160 @@
+
+Both methods require the following additional finance parameters to be specified:
+- `costing_generation_inflation` is used to adjust costs for each technology from the year provided under `discount_years` to the `cost_year` specified in the `plant_config` under the `plant` section.
+- `depreciation_method`: depreciation method to apply to capital items
+- `depreciation_period`: depreciation period (in years) for capital items (except electrolyzer, if used)
+- `depreciation_period_electrolyzer`: depreciation period (in years) for electrolyzer capital item (if used)
+
+(finance:direct_opt)=
+## Providing Finance Parameters: Direct Method
+Below is an example of the `finance_parameters` section of `plant_config` if financial parameters are input directly under the `finance_parameters` section.
+
+```yaml
+finance_parameters:
+  # Inflation parameters
+  profast_general_inflation: 0.0 # 0 for nominal analysis
+  costing_general_inflation: 0.025
+  # Finance parameters
+  discount_rate: 0.09
+  debt_equity_split: False
+  debt_equity_ratio: 2.62
+  property_tax: 0.02
+  property_insurance: 0.01
+  total_income_tax_rate: 0.257
+  capital_gains_tax_rate: 0.15
+  sales_tax_rate: 0.07375
+  debt_interest_rate: 0.07
+  debt_type: "Revolving debt"
+  load_period: 0
+  cash_onhand_months: 1
+  administrative_expense_percent_of_sales: 0.00
+  # Capital item depreciation parameters
+  depreciation_method: "MACRS"
+  depreciation_period: 5
+  depreciation_period_electrolyzer: 7
+  # Cost year of each component
+  discount_years:
+    wind: 2022
+    electrolyzer: 2022
+```
+
+This method also relies on data from `plant_config`:
+- `plant_life`: used as the `operating life` ProFAST parameter
+- `installation_time`: used as the `installation months` ProFAST parameter
+- `atb_year`: used to calculate the `analysis start year` ProFAST parameter where `analysis start year = atb_year + 2`
+
+
+```{note}
+`profast_general_inflation` is used to populate the escalation and inflation rates in ProFAST entries, a value of 0 correspondings to a *nominal analysis*
+```
+
+
+**Debt Equity Ratio vs Debt Equity Split**
+
+If `debt_equity_split` is True, then the value for `debt_equity_ratio` is assumed to be the `debt_equity_split` value. If `debt_equity_split` is False, the value for `debt_equity_ratio` is assumed to be the `debt_equity_ratio`. The relationship bewteen debt equity ratio and debt equity split is given below:
+
+$\text{debt equity ratio} = \frac{\text{debt equity split}}{100 - \text{debt equity split}}$
+
+$\text{debt fraction} = \frac{\text{debt equity ratio}}{\text{debt equity ratio + 1}}$
+
+$\text{equity fraction} = 1 - \text{debt fraction}$
+
+(finance:pf_params_opt)=
+## Providing Finance Parameters: ProFAST params config file
+
+```{note}
+ensure that ProFAST entries for `operating life` and `installation months` are the same as `plant_life` and `installation_time` specified in the `plant` section of the `plant_config` to avoid errors.
```

Comment:
The wording here is confusing. Can you more clearly group the paired terms that should be equal?